ntp/README.md

```diff
@@ -1,32 +1,57 @@
-# Ntp Integration
+# NTP check
 
-## Overview
+# Overview
 
-Get metrics from ntp service in real time to:
+Get alerts when your hosts drift out of sync with your chosen NTP server.
 
-* Visualize and monitor ntp states
-* Be notified about ntp failovers and events.
+# Installation
 
-## Installation
+The NTP check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) wherever you'd like to run the check. If you need the newest version of the check, install the `dd-check-ntp` package.
 
-Install the `dd-check-ntp` package manually or with your favorite configuration manager
+# Configuration
 
-## Configuration
+The Agent enables the NTP check by default, but if you want to configure the check yourself, create a file `ntp.yaml` in the Agent's `conf.d` directory:
 
-Edit the `ntp.yaml` file to point to your server and port, set the masters to monitor
+```
+init_config:
 
-## Validation
+instances:
+  - offset_threshold: 60 # seconds difference between local clock and NTP server when ntp.in_sync service check becomes CRITICAL; default is 60
+#   host: pool.ntp.org # set to use an NTP server of your choosen
```

Comment:
Actually, have all four of these commented-out options been deprecated? I don't see them in check.py.